### ISSUE TITLE
📱Mobile walkthrough improvement

### ISSUE THREAD

**Issue by kevincrepin** [citizenos/citizenos-fe#616]
For this enhancement, there's the requirement that this enhancement is already implemented: https://github.com/citizenos/citizenos-fe/issues/615

**The problem** is that the topic walkthrough card, also in collapsed state, is overlaying too much and vital content on mobile. This needs to be improved because it creates a complete mess:
![Screenshot 2024-01-09 at 11 32 51](https://github.com/citizenos/citizenos-fe/assets/26792281/9f4d2a77-54ec-4d8c-8ed9-5d31e47647e0)
Never mind the bottom menu, that's a separate issue.

My suggested solution is that on mobile, when the user closes the walkthrough start card or skips the tour, we hide it entirely, but since we now have the Help Centre in the main menu, we can give this a small highlight to make it more noticeable that something "is there". When opening the Help Centre in this context, we could show a specific section that "on this page, there's a walkthrough available", which the user could then initiate from the help centre.

Design suggestions:https://www.figma.com/file/gOJ4i00538e0qaP6IKIPVE/Platform-2.0-UI?type=design&node-id=11065%3A88515&mode=design&t=hMuZBFiOM13BzsmK-1
![Screenshot 2024-01-09 at 11 30 13](https://github.com/citizenos/citizenos-fe/assets/26792281/e508e264-843b-40e4-942a-745a36da8670)

**Comment by ilmartyrk** [comment]
@kevincrepin as the help-center is inside iframe I'll have to add the info box above the title "Citizen OS Help Center", but I guess it is OK. Otherwise I would have to start creating some kind of a cross-site solution trying to post data between iframe and app that could possibly become pain in the future

**Comment by kevincrepin** [comment]
Yes, that's fine :) Already great if we can have this context aware information in the first place!

**Comment by BeccaMelhuish** [comment]
@kevincrepin, @oksks pointed out that we're being a bit inconsistent in calling it a walkthrough versus a tour. Shall we keep it to 'tour'? (Tbh I think it's OK to mix it up a little to not be too repetitive within the same pop-up, but in this example, it's fully tour in one place and fully walkthrough in an other):

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/03b1a085-99f4-4e05-ad37-87ef0f2a73e4)

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/41fc88be-956b-4645-a297-afdcec2a45a6)

**Comment by kevincrepin** [comment]
Yes, let's use one "title" and maybe in some sentences it could still say "this tour is a walkthrough of features..." or smt if we still want to mix it up a notch.

I'll go over the designs and check for inconsistencies.
